pkg/controller/releasemanager/state.go

```diff
@@ -56,6 +56,11 @@ type Deployment interface {
 	sync(context.Context) error
 	retire(context.Context) error
 	del(context.Context) error
+	exitState(context.Context, string) error
```

Comment:
```suggestion
	exitState(ctx context.Context, state string) error
```

`context.Context` is clear on its own, but `string` isn’t